apps/editing-toolkit/editing-toolkit-plugin/full-site-editing-plugin.php

```diff
@@ -338,6 +338,10 @@ function load_tags_education() {
  * (Core Full Site Editing)
  */
 function load_wpcom_site_editor() {
+	// This is no longer needed after Gutenberg 12.1 due to the Navigation menu no longer being inscrutable.
+	if ( defined( 'GUTENBERG_VERSION' ) && version_compare( GUTENBERG_VERSION, '12.1.0', '>=' ) ) {
```

Comment:
This should be updated to `12.2.0`. I also thing that instead of conditional loading we can delete this code and folder after it lands 12.2 lands in production.

Reply:
Yup this should all be removed after `12.2.0` hits production. This is simply a mechanism to deploy for `edge` and `production` simultaneously.